cmd/networkcmd/upgrade.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
this is the same file as `pkg/constants/etnaDevnet/upgrade.json` no need to create another file here

Reply:
I believe etna one is going to be removed soon, so will keep this one